src/stemmabench/algorithms/stemma.py

```diff
@@ -85,14 +85,31 @@ def _set_folder_path(self, folder_path: str) -> None:
         if not os.path.isdir(folder_path):
             raise ValueError(f"{folder_path} is not an existing folder path.")
         self._folder_path = folder_path
-
-    def get_edges(self) -> None:
-        """Return array of all the edge values.
-
-        ### Raises:
-            - NotImplementedError: Not implemented yet.
+    
+    def get_edge_values(self) -> Dict[str, float]:
+        """Return dictionary with edges as keys an edge distances as values.
+        
+        ### Returns:
+            - dict: Dictionary with edges as keys and edge distances as values.
         """
-        raise NotImplementedError
+        out = {}
+        for key in self.text_lookup:
+            print(f"key: {key}, edges: {self.text_lookup[key].edges}")
+            for i in range(len(self.text_lookup[key].edges)):
+                out.update({self.text_lookup[key].label + "," + self.text_lookup[key].children[i].label: self.text_lookup[key].edges[i]})
```

Comment:
f-string would also be clearer here

``
```suggestion
                out.update(f"{self.text_lookup[key].label} ,{self.text_lookup[key].children[i].label}": self.text_lookup[key].edges[i]})
```

Reply:
F-string added